share/spack/qa/run-build-tests

```diff
@@ -41,4 +41,4 @@ fi
 spack config get compilers
 
 # Run some build smoke tests, potentially with code coverage
-${coverage_run} bin/spack install ${OPTIONS} ${SPEC}
+${coverage_run} bin/spack -d install ${OPTIONS} ${SPEC}
```

Comment:
do we want to make this change?  I think at the very least it doesn't belong in this PR.  I'm assuming that this was some vestigial change for debugging Travis.